nservicebus/event-streaming/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Event stream processing
+summary: Combining the Particular Service Platform with an event stream processor (Apache Kafka, Azure Event Hubs, or Amazon Kinesis)
+callsToAction:
+  - interest-apache-kafka
+  - interest-azure-event-hubs
+  - interest-amazon-kinesis
+reviewed: 2024-02-06
+related:
+  - samples/azure-functions/service-bus-kafka
+---
+
```

Comment:
It would be nice and probably more engaging to have a nice picture of Kafka.
In my previous attempt at this, I went with:
![Kafka logo](https://apache.org/logos/res/kafka/kafka_highres.png)

Reply:
I took it out because it looked odd, especially with the callouts. I tried moving it around to make it fit right and it causes a bunch of misalignment.